packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -101,14 +104,19 @@ export function PanelChrome({
   const containerStyles: CSSProperties = { width, height };
   const ariaLabel = title ? selectors.components.Panels.Panel.containerByTitle(title) : 'Panel';
 
+  // Shift the hover menu down if it's on the top row so it doesn't get clipped by topnav
+  const yOffset = (gridPos?.y ?? 0) === 0 ? -16 : undefined;
+
   return (
     <div className={styles.container} style={containerStyles} aria-label={ariaLabel}>
+      {!hasHeader && menu && <HoverWidget menu={menu} title={title} offset={yOffset} />}
+
       <div className={styles.loadingBarContainer}>
         {loadingState === LoadingState.Loading ? <LoadingBar width={'28%'} height={'2px'} /> : null}
       </div>
 
       <div className={styles.headerContainer} style={headerStyles} data-testid="header-container">
-        {title && (
+        {title && hasHeader && (
```

Comment:
I think `hasHeader` should wrapper the whole `div headerContainer`. We should take out the `statusMessage condition` out then.
Because the description, links, etc. should also show up in the hover header box, unlike:
![image](https://user-images.githubusercontent.com/13227501/213734904-2fee4603-8e59-422d-870c-de5ab1c4af37.png)
![image](https://user-images.githubusercontent.com/13227501/213734957-ede4df9b-9d07-4ca9-b8da-c9f4185eb1c8.png)